metadata-models/src/main/pegasus/com/linkedin/metadata/search/DatasetDocument.pdl

```diff
@@ -72,4 +72,24 @@ record DatasetDocument includes BaseDocument {
    * List of tags for this dataset
    */
   tags: optional array[string]
+
+  /**
+   * List of field descriptions
+   */
+  fieldDescriptions: optional array[string]
+
+  /**
+   * List of tags applied to fields
+   */
+  fieldTags: optional array[string]
+
+  /**
+   * List of field descriptions
+   */
+  editedFieldDescriptions: optional array[string]
```

Comment:
My understanding is that `fieldDescriptions` is coming from `SchemaMetadata`, whereas `editedFieldDescriptions` is coming from `EditableSchemaMetadata`. But what is the source of truth when it comes to field description? If a user has made changes to a field description via UI, shouldn't that be the "updated" field description? Why do we need to keep both?
Not sure if this has been discussed before. If yes, please feel free to point to relevant doc/rfc.

Reply:
That's a good question @jywadhwani 
I'm inclined to let this implementation stay for the following reasons: 
1. We should keep the "human written" documentation separate from the "schema-attached" documentation (we are already doing that in the db, so makes sense to continue doing it in the indexes. This way, we don't mix the two streams of information (especially without a governed approval flow for documentation overrides). 
2. Once we build that sort of approval flow in the application, we can store breadcrumbs for when an editable schema field description overrides an existing schema field description (imagine a flag on the schema-field-description marking it as not-to-be-used), which would trigger the indexer to remove this text from the index. 

Hope this addresses your concern.

Reply:
Sounds good, thanks for the explanation @shirshanka.